mbtiles/cf.py

```diff
@@ -0,0 +1,49 @@
+# concurrent.futures implementation
+
+import concurrent.futures
+from itertools import islice
+import logging
+
+from mbtiles.worker import init_worker, process_tile
+
+BATCH_SIZE = 100
+
+log = logging.getLogger(__name__)
+
+
+def process_tiles(
+    conn,
+    tiles,
+    insert_results,
+    num_workers=None,
+    inputfile=None,
+    base_kwds=None,
+    resampling=None,
+    img_ext=None,
+    image_dump=None,
+):
+    with concurrent.futures.ProcessPoolExecutor(
```

Comment:
I haven't done any more than play with `concurrent.futures`, so I can't really review this at all effectively.